### ISSUE TITLE
Discord /tts (text to speech) function not support

### ISSUE THREAD

**Issue by dklug** [Half-Shot/matrix-appservice-discord#103]
Currently the Discord text to speech function is not supported.

![image](https://user-images.githubusercontent.com/8633205/36931901-d7740730-1e74-11e8-8dbc-7b3af19d74e2.png)

It should be supported

**Comment by Half-Shot** [comment]
I'm assuming when you send the command. Nothing happens. In that case it probably needs sending via the bot. Will take a look later

**Comment by dklug** [comment]
Yeah, additionally clients like Riot will get confused with the command /tts, we'll need a new command like !discord_tts

Looks like tts is just a bool in the message object:
[https://discordapp.com/developers/docs/resources/channel#create-message](url)
